src/components/Pagination/Pagination.tsx

```diff
@@ -0,0 +1,41 @@
+import ReactPaginate from "react-paginate";
+import "./Pagination.css";
+
+function Pagination({
+  pageCount,
+  handlePageClick,
+  currentPage,
+}: {
+  pageCount: number;
+  handlePageClick: (selectedItem: { selected: number }) => void;
+  currentPage: number;
+}) {
+  return (
+    <>
+      {pageCount > 0 && (
+        <ReactPaginate
+          pageCount={pageCount}
+          previousLabel={`<<`}
+          nextLabel={`>>`}
+          breakLabel={`...`}
+          marginPagesDisplayed={2}
+          pageRangeDisplayed={3}
+          onPageChange={handlePageClick}
+          forcePage={currentPage}
+          containerClassName={`pagination justify-content-center`}
+          pageClassName={`page-item`}
+          pageLinkClassName={`page-link`}
```

Comment:
do you really need to duplicate all these items and links?

Reply:
Would those classes be spread automatically to previous and next properties if you pass only general one?